src/pages/SignMessage/components/SignSuccess.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { faCheck } from '@fortawesome/free-solid-svg-icons';
+import { useSignMessage } from '@multiversx/sdk-dapp/hooks/signMessage';
+import { PageState } from '@multiversx/sdk-dapp/UI';
+import { CopyButton } from '@multiversx/sdk-dapp/UI/CopyButton';
+import { Link } from 'react-router-dom';
+import { routeNames } from 'routes';
+import { SignMessageResultPropsType } from '../types';
+
+export const SignSuccess = ({ message }: SignMessageResultPropsType) => {
```

Comment:
why not select message from sdk-dapp, since you know it's success, then SignMessageResultPropsType would no longer be needed